services/src/grpc-speaker/grpc-api/src/main/java/org/openkilda/grpc/speaker/model/PortConfigDto.java

```diff
@@ -0,0 +1,62 @@
+/* Copyright 2019 Telstra Open Source
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+
+package org.openkilda.grpc.speaker.model;
+
+import org.openkilda.messaging.model.grpc.OnOffState;
+import org.openkilda.messaging.model.grpc.PortMode;
+import org.openkilda.messaging.model.grpc.PortPause;
+import org.openkilda.messaging.model.grpc.PortSpeed;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.databind.PropertyNamingStrategy.SnakeCaseStrategy;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+
+@Data
```

Comment:
the same

Reply:
In this place Value annotation can't not applied